src/server/game/Entities/Item/Item.cpp

```diff
@@ -962,6 +962,17 @@ bool Item::GemsFitSockets() const
     return true;
 }
 
+bool Item::HasAnySocket() const
```

Comment:
Personally I'd name it HasSocket
Why I don't know, just feel like the "any" is kinda useless :D

Reply:
I agree with @Kitzunu

Reply:
Agree 🐱 
IsAlive();
IsPet();
HasAura();
IsHeroic();
etc etc etc.. already on core.